src/Dockerfile

```diff
@@ -1,6 +1,10 @@
 FROM amsterdam/docker_python:latest
 MAINTAINER datapunt@amsterdam.nl
 
+RUN pip3 install --upgrade pip
+RUN mkdir ~/.ssh
```

Comment:
I would recommend including some comments why this is required

Reply:
Removed it, it was not necessary